j2lint/rules/jinja_statement_has_space_rule.py

```diff
@@ -27,7 +27,7 @@ def check(self, line):
         Returns:
             Object: Returns error object if found else None
         """
-        logger.debug("Check line rule does not exist for %s",
+        logger.debug("Check text rule does not exist for %s",
```

Comment:
I don't understand why you state `does not exist` - how do you know if does not exist here? the test is done afterin the regex. Am I missing something?

Reply:
I had added the logs as some rules are having check method and some are having checktext, but now as we are defining the default methods in rule.py  I think we can remove these logs.